src/app/services/views/ServiceTrayView.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <RouteView name="service-summary-view">
+    <AppView>
+      <template #title>
+        <div class="tray-title-wrapper">
+          <img
+            aria-hidden="true"
+            src="@/assets/images/icon-wifi-tethering.svg?url"
+          >
+
+          <h2 class="tray-title">
+            <RouterLink
+              :to="{
+                name: 'service-detail-view',
+                params: {
+                  service: props.data.name,
+                },
+              }"
+            >
+              <RouteTitle
+                :title="t('services.routes.item.title', { name: props.data.name })"
+                :render="true"
+              />
+            </RouterLink>
+          </h2>
+        </div>
+      </template>
+
+      <div class="stack">
+        <div>
+          <h3>{{ t('services.overview') }}</h3>
+
+          <div class="mt-4">
```

Comment:
How come `stack` spacing isn't being applied here, is it because of that extra div I was asking about?

Reply:
See https://github.com/kumahq/kuma-gui/pull/1611#discussion_r1367041210.